scripts/tasks.sh

```diff
@@ -479,7 +479,7 @@ function taint_primaries() {
 
     # Rook tolerations
     if kubectl get namespace rook-ceph &>/dev/null; then
-        kubectl -n rook-ceph patch cephclusters rook-ceph --type=merge -p '{"spec":{"placement":{"all":{"tolerations":[{"key":"node-role.kubernetes.io/master","operator":"Exists"}]}}}}'
+        kubectl -n rook-ceph patch cephclusters rook-ceph --type=merge -p '{"spec":{"placement":{"all":{"tolerations":[{"key":"node-role.kubernetes.io/control-plane","operator":"Exists"}]}}}}'
```

Comment:
Rather than remove the existing `node-role.kubernetes.io/master` toleration can we append `node-role.kubernetes.io/control-plane` as an additional toleration?